stix_shifter/stix_transmission/src/modules/splunk/spl_api_client.py

```diff
@@ -75,7 +75,7 @@ def get_search_results(self, search_id, offset, count):
             data['offset'] = str(offset)
             data['count'] = str(count)
         # response object body should contain information pertaining to search.
-        return self.client.call_api(endpoint, 'GET', data=data)
```

Comment:
both ways work to me, they just fetch different amount of data.
The question is which way is correct?
for query: 'SELECT QIDNAME(qid) as qidname, qid as qid, CATEGORYNAME(category) as categoryname, category as categoryid, CATEGORYNAME(highlevelcategory) as high_level_category_name, highlevelcategory as high_level_category_id, logsourceid as logsourceid, LOGSOURCETYPENAME(logsourceid) as logsourcetypename, LOGSOURCENAME(logsourceid) as logsourcename, starttime as starttime, endtime as endtime, devicetime as devicetime, sourceaddress as sourceip, sourceport as sourceport, sourcemac as sourcemac, destinationaddress as destinationip, destinationport as destinationport, destinationmac as destinationmac, username as username, eventdirection as direction, identityip as identityip, identityhostname as identity_host_name, eventcount as eventcount, PROTOCOLNAME(protocolid) as protocol, BASE64(payload) as payload, URL as url, magnitude as magnitude, Filename as filename, URL as domainname FROM events WHERE url = '"'"'www.example.com'"'"' limit 10000 last 5 minutes'
parameters in body (before fix) way returns 94 data items;
parameters in url (after fix) way returns 10 data items.

Reply:
the limit is 10000 (this means its a total of 10000 results in the search id cursor)
but when we go to fetch the 10000 items we want to fetch them in sizes of 1000 at a time.  So we pass 0 to 1000 for first 1000, then we do 1001 to 2000 to get the next 1000.